### ISSUE TITLE
Reopening Export Template Manager when template is being downloaded results in errors

### ISSUE THREAD

**Issue by kleonc** [godotengine/godot#55365]
### Godot version

3.4.stable

### System information

Windows 7

### Issue description

Title. Errors:
```
ERROR: Condition "get_http_client_status() != HTTPClient::STATUS_DISCONNECTED" is true.
   at: set_download_file (scene/main/http_request.cpp:428)
ERROR: Condition "get_http_client_status() != HTTPClient::STATUS_DISCONNECTED" is true.
   at: set_use_threads (scene/main/http_request.cpp:409)
ERROR: HTTPRequest is processing a request. Wait for completion or cancel it before attempting a new one.
   at: (scene/main/http_request.cpp:82)

```

### Steps to reproduce

1. Open Export Template Manager and start downloading a template.
2. Close Export Template Manager.
3. Open Export Template Manager, after a while it will produce the errors (if still downloading).

### Minimal reproduction project

N/A

**Comment by andreymal** [comment]
This bug is still present in 4.2.stable

**Comment by kleonc** [comment]
Fixed by #93148 (since v4.3.beta2.official [b75f0485b]).
